students/jesse_miller/session02/fibonacci_test.py

```diff
@@ -0,0 +1,19 @@
+#!/usr/local/bin/python3
+"""This is a test to make sure I understand how the code works."""
+import sys
+"""import sys for .argv input"""
+n = int(sys.argv[1])
```

Comment:
You shouldn't use `sys.argv` for these assignments as taking in user input and validating it and so forth is a good learning experience.